Rakefile

```diff
@@ -13,6 +13,28 @@ rescue ::LoadError
   require 'yaml'
 end
 
+if Gem.win_platform?
```

Comment:
👍

Reply:
Thanks.  That's why I built https://msp-greg.github.io/, so I could relax with a tablet and find that stuff...